keeper/src/main/scala/zio/keeper/membership/ByteCodec.scala

```diff
@@ -17,9 +19,9 @@ object ByteCodec {
     new ByteCodec[A] {
 
       def toChunk(a: A) =
-        ZIO.effect(Chunk.fromArray(writeBinary(a)(rw))).mapError(SerializationError("Failed serializing to chunk.", _))
+        ZIO.effect(Chunk.fromArray(writeBinary(a)(rw))).mapError(SerializationTypeError(_))
```

Comment:
I don't know whether calling this a TypeError is a good idea

Reply:
why?

Reply:
I will changed that